src/controllers/v2/renewal.js

```diff
@@ -140,46 +151,41 @@ const RenewalController = {
       }
     }
 
+    let renewalRegistration;
+
     await db.sequelize.transaction(async (t) => {
       try {
-        await RegistrationHistory.create(
-          {
-            ...mappedExistingReg,
-            RegistrationId: registrationNumber,
-            createdByLicensingOfficer: existingReg.createdByLicensingOfficer,
-            // Need to confirm expected behaviour here!
-            expiryDate: existingReg.expiryDate
-          },
+        const initialRegistration = await Registration.findByPk(registrationNumber);
```

Comment:
Is this not duplicate of line 104? `const existingReg = findOneRegistrationResponse;`